Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -4,26 +4,38 @@ import PayPalWebCheckout
 import PaymentsCore
 import AuthenticationServices
 import PayPalNativeCheckout
+import PayPalCheckout
 
 /// This class is used to share the orderID across shared views, update the text of `bottomStatusLabel` in our `FeatureBaseViewController`
 /// as well as share the logic of `processOrder` across our duplicate (SwiftUI and UIKit) card views.
 class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate, PayPalDelegate {
+    func paypalDidShippingAddressChange(shippingChange: ShippingChange, shippingChangeAction: ShippingChangeAction) {
```

Comment:
I agree that separating the delegate conformance to an extension would make things a bit clearer to merchants. Not sure if separating into new files would make it as clear to them though. I feel like extensions in the same file would be enough since it's only a 300 line file? That way merchants don't have to go hunting around.

Reply:
True true - this class has kinda been a catch all, so I am fine with whatever allows us to clean this file up a bit for merchants to see how to do things per feature vs having to sift through this file and pick and choose what they need.

Reply:
@jaxdesmarais reflecting more on this, I king of don't think we should move them if we think this file will stay under 500 lines, as that's the general ideal max length of a file on most xcode linters. Seems like a win to make it so that merchants can easily see working examples in one place as long as they are organized. But, if we are going to be continuously conforming to more delegates with this file, then I think you're right and it might be best to just start breaking them out now.

Reply:
That makes sense to me! For now, I think organizing this file with `MARK`s for each feature would go a long way.

Reply:
I agree that this file has a lot of responsibility, but I think it's fine for now, as it isn't too long and our demo app is pretty simple. 

Organizing it by pragma mark (`// MARK: - <PROTOCOL_NAME>`) or class extensions within the file both sound like good options to me!

Reply:
this class does have lot of responsibility will go with `//Mark:` separation now, but we can limited in base and create viewModels for different views later